### PR TITLE
[TIDOC-2435] Adjust live photo docs

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA's: 
- https://jira.appcelerator.org/browse/TIDOC-2435
- https://jira.appcelerator.org/browse/TIMOB-20486

### CODE REVIEW

File: apidoc/Titanium/UI/iOS/LivePhotoView.yml [hunk 1]
```diff
@@ -14,8 +14,19 @@ extends: Titanium.UI.View
 platforms: [iphone,ipad]
 since: "5.2.0"
 osver: {ios:{min: "9.1"}}
+excludes:
+    events: [click,dblclick,doubletap,longpress,pinch,postlayout,singletap,swipe,
```
Comment (FokkeZB) [inline apidoc/Titanium/UI/iOS/LivePhotoView.yml]: Is there no way to exclude all?

**Comment by FokkeZB** [comment]
PR verified (assume you've validated doc yourself) to address all comments

File: apidoc/Titanium/UI/iOS/LivePhotoView.yml (hunk 1 above)
Comment (hansemannn) [inline apidoc/Titanium/UI/iOS/LivePhotoView.yml]: Didn't find anything. The above example is the same as the Ti.UI.ListView uses for example.

**Comment by hansemannn** [comment]
Of course :rocket:

File: apidoc/Titanium/UI/iOS/LivePhotoView.yml (hunk 1 above)
Comment (FokkeZB) [inline apidoc/Titanium/UI/iOS/LivePhotoView.yml]: guess not:

- https://github.com/appcelerator/tiduck/blob/fa447ee5bd95189d2dd8c8647be4f2252a17c545/apidoc/docgen.py#L371
- https://github.com/appcelerator/tiduck/blob/fa447ee5bd95189d2dd8c8647be4f2252a17c545/apidoc/generators/jsduck_generator.py#L418

**Pull request merged**